.travis/deploy.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+tag_version=$1
+
+mkdir ~/.bintray/
+FILE=$HOME/.bintray/.credentials
+cat <<EOF >$FILE
+realm = Bintray API Realm
+host = api.bintray.com
+user = $BINTRAY_USER
+password = $BINTRAY_API_KEY
+EOF
+
+cd $TRAVIS_BUILD_DIR
+pwd
+
+project_version=$(sbt version -Dsbt.log.noformat=true | perl -ne 'print $1 if /(\d+\.\d+[^\r\n]*)/')
```

Comment:
It seems there is a problem with this line, since locally it gives 0.13/plugins0.5.0-M1